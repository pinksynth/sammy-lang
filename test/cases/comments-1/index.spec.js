const { performSnapshotAssertions } = require("../../helpers")
performSnapshotAssertions("Comments", __dirname)
