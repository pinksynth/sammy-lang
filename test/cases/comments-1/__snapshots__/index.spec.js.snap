// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ“¸ Comments 1`] = `
[
  {
    "columnNumberEnd": 1,
    "columnNumberStart": 1,
    "lineNumberEnd": 3,
    "lineNumberStart": 1,
    "tokenType": "COMMENT",
    "value": "# Inline comment

",
  },
  {
    "columnNumberEnd": 4,
    "columnNumberStart": 1,
    "lineNumberEnd": 5,
    "lineNumberStart": 3,
    "tokenType": "COMMENT",
    "value": "<<<
  Multiline comment
>>>",
  },
]
`;

exports[`ðŸ“¸ Comments 2`] = `
{
  "children": [],
  "type": "ROOT",
}
`;

exports[`ðŸ“¸ Comments 3`] = `""`;
